extension/intents/scroll/scrollHelper.js

```diff
@@ -3,7 +3,13 @@
 const scrollAmount = 0.9;
 
 function scrollVertically(dy, smooth) {
-  window.scrollBy({
+  let toBeScrolled = window;
+
+  if (document.contentType === "application/pdf") {
```

Comment:
Really elegant way to solve the problem.  

My only thought is. There can be many other contents as well that can be covered by your code (in addition to pdf). What if we put add html related contentType with **window** and ```toBeScrolled = document.activeElement``` for remaining.